app/main.py

```diff
@@ -2,5 +2,14 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    saved_runs = {}
+
+    def wrapper(*args) -> (int, float):
+        if args in saved_runs.keys():
```

Comment:
The check if args in saved_runs is more efficient than if args in saved_runs.keys(), as checking for the presence of a key in a dictionary by default checks the keys.